scroll-animations-1/Overview.bs

```diff
@@ -471,18 +475,26 @@ one the following values:
 The procedure to <dfn>resolve a container-based offset</dfn> given
 <var>offset</var> is as follows:
 
-1.  [=effective scroll offset=] is the scroll offset corresponding to the first
-    matching condition from the following:
+1.  If <em>any</em> of the following are true:
+
+    * {{source}} is null, or
+    * {{source}} does not currently have a [=CSS layout box=], or
+    * {{source}}'s layout box is not a [=scroll container=].
+
+    The [=effective scroll offset=] is null.
+
+1.  The [=effective scroll offset=] is the scroll offset corresponding to the
+    first matching condition from the following:
```

Comment:
This seems odd. Should the second step say, "Otherwise, ..." ? Or should the first step return early?

Reply:
Good catch. Fixed.